src/utils/runWithTruffle.js

```diff
@@ -1,19 +1,29 @@
-import Web3 from 'web3'
-import TruffleConfig from 'truffle-config'
-import TruffleCompile from 'truffle-workflow-compile'
-import TruffleEnvironment from 'truffle-core/lib/environment'
-import { Logger } from 'zos-lib'
+import { FileSystem as fs, Logger } from 'zos-lib'
 
 const log = new Logger('RunWithTruffle')
-const options = { logger: console }
-const config = TruffleConfig.detect(options)
+
+export default function runWithTruffle(script, network, compile = false) {
+  const TRUFFLE_JS_PATH = `${process.cwd()}/truffle.js`
+  const TRUFFLE_CONFIG_JS_PATH = `${process.cwd()}/truffle-config.js`
+
+  const truffleFileMissing = !fs.exists(TRUFFLE_JS_PATH) && !fs.exists(TRUFFLE_CONFIG_JS_PATH);
```

Comment:
Note that truffle does not require a truffle.js to be present in the working dir, but to be present _anywhere up in the docs hierarchy_ (see [here](https://github.com/trufflesuite/truffle-config/blob/develop/index.js#L236)). To ensure we are consistent with truffle, can't we use TruffleConfig.detect here as well, and catch the exception if the file is not found (or just let it bubble up)?

Reply:
Agree, let's do that